src/retry_failed.py

```diff
@@ -117,7 +117,8 @@ def main(args):
         NB_CONSUMERS = 0
```

Comment:
I had to edit line 91 to `    wllt_clnt_mngr = WalletClientManager(client_path, args.node_addr, contracts_by_alias, addresses_by_pkh, managers,
                                         verbose=args.verbose)`
Except for that, it looks good to me and with this little change it worked fine for me.

Reply:
Nice catch.